### ISSUE TITLE
Is step1 complete? got message: "reading *_unaligned_R2.fastq not such file or directory"

### ISSUE THREAD

**Issue by bmunozsa** [USDA-VS/snp_analysis#12]
Hello, I've run vSNP step 1, and the program stopped after outputting this message "reading *_unaligned_R2.fastq not such file or directory". Why does this happens? Has the step1 finished correctly anyway?

Thanks in advance,

**Comment by stuber** [comment]
It may not be finding the input file.  Is this at input?  If possible send a screen shot showing working directory contents and command being used.  

May try step 1 from a working directory containing just one sample (a single set of paired files)
`vSNP_step1.py -r1 *_R1*.fastq.gz -r2 *_R2*.fastq.gz`

**Comment by bmunozsa** [comment]
Thanks for finding the time to help me. Of course, this a screen shot from the folder before execution of the following line

_**vSNP_step1.py -r1 LOBO1_S6_L001_R1_001.fastq.gz -r2 LOBO1_S6_L001_R2_001.fastq.gz -r AY651334.fasta**_

![step1_folder_before_exec](https://user-images.githubusercontent.com/124638641/222776352-9daad9a4-2fb3-4841-95b2-13c5adc92d11.png)

and this is the message when the program stops
![error message](https://user-images.githubusercontent.com/124638641/222793729-df5beddb-7630-4823-9d42-94e875d436f7.png)

while this is the state of that same folder by that moment
![state 2](https://user-images.githubusercontent.com/124638641/222794138-6a4b579e-a5e3-4716-92eb-28116b8e7283.png)

**Comment by bmunozsa** [comment]
> It may not be finding the input file. Is this at input? If possible send a screen shot showing working directory contents and command being used.
> 
> May try step 1 from a working directory containing just one sample (a single set of paired files) `vSNP_step1.py -r1 *_R1*.fastq.gz -r2 *_R2*.fastq.gz`

Do you think it would be useful to also share the actual files with you?

**Comment by stuber** [comment]
Yes, feel free to send me a link to the files at tod.p.stuber@usda.gov.
